tests/k8s_cmdline_tests.yaml

```diff
@@ -422,3 +422,80 @@
     --pod_list helm_tests/helm_test6/
     --expected_output expected_cmdline_output_files/helm_test_resolved_yaml_next_to_multi_charts.txt
   expected: 0
+
+- name: basic_connectivity_specific_nodes-expl
+  args: >
+    --connectivity k8s_testcases/example_policies/testcase8/testcase8-networkpolicy1.yaml
+    --ns_list k8s_testcases/example_podlist/ns_list.json
+    --pod_list k8s_testcases/example_podlist/pods_list.json
+    --explain default/cog-agents-d54st,default/cog-agents-qr8gp 
```

Comment:
in the documentation you specify args to explain as deploymnent names (such as `default/deployment-A1`), not pod names, but here you use pod names.. so which one is the correct format?

Reply:
the user can use node names (both deployments or pods).

Reply:
so this should also be specified in the doc.

Reply:
I mean, the type of the node is not relevant (that is why I used 'node'). I have diversified the example to include both a pod and a deployment to make it more clear.